fastly/custom_tls_certificate.go

```diff
@@ -0,0 +1,195 @@
+package fastly
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/google/jsonapi"
+)
+
+// CustomCertificate represents a custom certificate. Uses common TLSDomain type from BulkCertificate
+type CustomCertificate struct {
```

Comment:
To follow convention with other structures in this PR, this should be renamed to `CustomTLSCertificate` and the corresponding CRUD methods should also be renamed to match.
```suggestion
// CustomTLSCertificate represents a custom certificate. Uses common TLSDomain type from BulkCertificate.
type CustomTLSCertificate struct {
```

Reply:
Updated